pagopa/pn-frontend
# fix/PN-13073 - fix the order of legal contacts types

## Short description
this PR sets the order of legal contacts types inside the ContactsSummaryCards component for both PG and PF web-apps so that the first contact type to be displayed is the one associated with the default sender.

## How to test
Login on both PF and PG webapp and verify the first legal contact type displayed inside the ContactsSummaryCards component is always the one associated with the default legal contact.
